commands/rm.go

```diff
@@ -34,10 +34,11 @@ func cmdRm(c *cli.Context) {
 		if err := mcn.Remove(host, force); err != nil {
 			log.Errorf("Error removing machine %s: %s", host, err)
 			isError = true
+		} else {
+			log.Infof("Successfully removed %s", host)
```

Comment:
Hm, so in my opinion we do not really need that little `if isError` block at the end since the users will be notified if the removal fails, so maybe we can take that out with the PR too?  cc @ehazlett WDYT?

Also, eventually we will need to move over this function to the new method of doing the actions concurrently (that `runActionWithContext` thing), since this way is old (it was the first multiarg of its kind).  But that can be done in another PR.